react-bootstrap/react-bootstrap.d.ts

```diff
@@ -1071,5 +1071,60 @@ declare module "react-bootstrap" {
     }
     class Position extends React.Component<PositionProps, {}> {
     }
+    // <Media.Left />
+    interface MediaLeftProps {
+      align?: string;
+    }
+    type MediaLeft = React.ClassicComponent<MediaLeftProps, {}>;
+    var MediaLeft: React.ClassicComponentClass<MediaLeftProps>;
+
+    // <Media.Right />
+    interface MediaRightProps {
+      align?: string;
+    }
+    type MediaRight = React.ClassicComponent<MediaRightProps, {}>;
+    var MediaRight: React.ClassicComponentClass<MediaRightProps>;
+
+    // <Media.Heading />
+    interface MediaHeadingProps {
+      componentClass?: React.ReactType;
+    }
+    type MediaHeading = React.ClassicComponent<MediaHeadingProps, {}>;
+    var MediaHeading: React.ClassicComponentClass<MediaHeadingProps>;
+
+    // <Media.Body />
+    interface MediaBodyProps {
+      componentClass?: React.ReactType;
+    }
+    type MediaBody = React.ClassicComponent<MediaBodyProps, {}>;
+    var MediaBody: React.ClassicComponentClass<MediaBodyProps>;
+
+    // <Media.List />
+    interface MediaListProps {
+    }
+    type MediaList = React.ClassicComponent<MediaListProps, {}>;
+    var MediaList: React.ClassicComponentClass<MediaListProps>;
+
+    // <Media.ListItem />
+    interface MediaListItemProps {
+      componentClass?: React.ReactType;
```

Comment:
+1, some where in the file you are using 2 spaces, and 4 spaces elsewhere. Please be consistent.